dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Accelerate module for PyOpenGL"
+HOMEPAGE="http://pyopengl.sourceforge.net/ https://pypi.org/project/PyOpenGL-accelerate/"
+MY_PN="PyOpenGL-accelerate"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	>=dev-python/sphinx-2.0.1-r1[${PYTHON_USEDEP}]
+	>=dev-python/sphinx-epytext-0.0.4[${PYTHON_USEDEP}]
+	>=dev-python/genshi-0.7.3-r1[${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.18.2[${PYTHON_USEDEP}]
+	>=dev-python/pydocstyle-5.0.2[${PYTHON_USEDEP}]
+	>=dev-python/pyopengl-3.1.5[${PYTHON_USEDEP}]
+	>=dev-python/six-1.14.0[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
```

Comment:
Move this to under `SRC_URI`

Does this package not have tests? If it has tests then they should be enabled in the ebuild. If the tests can't be run because they are incompatible with sandbox, or need hardware, then use `RESTRICT="test"` with a comment explaining why.

Reply:
I switched to github for the download (you need a commit hash since they don't have tags) and the tests need a video card to run.

```
==================================== ERRORS ====================================
_________________ ERROR collecting tests/test_arraydatatype.py _________________
tests/test_arraydatatype.py:1: in <module>
    import basetestcase
tests/basetestcase.py:17: in <module>
    pygame.display.init()
E   pygame.error: No available video device
------------------------------- Captured stdout --------------------------------
pygame 1.9.6
Hello from the pygame community. https://www.pygame.org/contribute.html
_____________________ ERROR collecting tests/test_core.py ______________________
tests/test_core.py:19: in <module>
    pygame.display.init()
E   pygame.error: No available video device
__________________ ERROR collecting tests/test_evaluators.py ___________________
tests/test_evaluators.py:1: in <module>
    import basetestcase
tests/basetestcase.py:17: in <module>
    pygame.display.init()
E   pygame.error: No available video device
_____________________ ERROR collecting tests/test_tess.py ______________________
tests/test_tess.py:1: in <module>
    import basetestcase
tests/basetestcase.py:17: in <module>
    pygame.display.init()
E   pygame.error: No available video device
___________________ ERROR collecting tests/test_textures.py ____________________
tests/test_textures.py:1: in <module>
    import basetestcase
tests/basetestcase.py:17: in <module>
    pygame.display.init()
E   pygame.error: No available video device
=========================== short test summary info ============================
ERROR tests/test_arraydatatype.py - pygame.error: No available video device
ERROR tests/test_core.py - pygame.error: No available video device
ERROR tests/test_evaluators.py - pygame.error: No available video device
ERROR tests/test_tess.py - pygame.error: No available video device
ERROR tests/test_textures.py - pygame.error: No available video device
!!!!!!!!!!!!!!!!!!! Interrupted: 5 errors during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 skipped, 5 errors in 5.76s =========================
 * ERROR: dev-python/pyopengl_accelerate-3.1.5::gentoo failed (test phase):
```